taxi/urls.py

```diff
@@ -18,7 +18,10 @@
     ),
     path("cars/", CarListView.as_view(), name="car-list"),
     path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
-    path("drivers/", DriverListView.as_view(), name="driver-list"),
+    path("drivers/",
+         DriverListView.as_view(),
+         name="driver-list"
```

Comment:
There is a minor formatting issue here. The path definition for 'drivers/' is split across multiple lines, which is fine, but it should be consistent with the rest of the file. Consider placing the `name` parameter on the same line as the `path` function call for consistency, similar to the 'cars/' path definition.